README.md

```diff
@@ -147,28 +162,31 @@ Find **Protocol Buffers Descriptions** at the [`./pb` directory](./pb).
     are seeing this, run `kubectl get service frontend-external -o=yaml | kubectl apply -f-`
     to trigger load balancer reconfiguration.
 
-<!-- 
-### Option 3: Using Pre-Built Container Images 
+### Option 3: Using Pre-Built Container Images
+
+> 💡 Recommended if you want to deploy the app faster in fewer steps to an
+> existing cluster.
 
-> 💡 Recommended for test-driving the application on an existing cluster. 
+Follow "Option 1" or "Option 2" until you reach the `skaffold run` step. Instead
+of building/uploading container images, you can use the pre-built images by
+deploying the [release manifest](./release) below.
 
-**Prerequisite**: a running Kubernetes cluster. 
+**Prerequisite**: a running Kubernetes cluster (either local or on cloud).
 
-1. Clone this repository.
-1. Deploy the application: `kubectl apply -f ./release/kubernetes-manifests`  
-1. Run `kubectl get pods` to see pods are in a healthy and ready state.
+1. Clone this repository, and go to the repository directory
+1. Run `kubectl apply -f ./release/kubernetes-manifests` to deploy the app.
+1. Run `kubectl get pods` to see pods are in a Ready state.
 1.  Find the IP address of your application, then visit the application on your
     browser to confirm installation.
 
-        kubectl get service frontend-external
--->
+        kubectl get service/frontend-external
 
 ### (Optional) Deploying on a Istio-installed GKE cluster
 
 > **Note:** you followed GKE deployment steps above, run `skaffold delete` first
 > to delete what's deployed.
 
-1. Create a GKE cluster (described above).
+1. Create a GKE cluster (described in "Option 2").
```

Comment:
link to the header for Option 2?

Reply:
that's very hard to do since titles are changing. I don't think gfm lets us create anchor IDs.